.github/workflows/security.yml

```
@@ -59,6 +59,29 @@ jobs:
           path: /home/runner/vuln-cache
           key: trivy-cache-${{ github.run_id }}
 
+      - name: Get KEV list
+        run: |
+          cd /home/runner
+          curl -s -o kev.json https://www.cisa.gov/sites/default/files/feeds/known_exploited_vulnerabilities.json
```

```suggestion
          curl -s --compressed -o kev.json https://www.cisa.gov/sites/default/files/feeds/known_exploited_vulnerabilities.json
```

Downloads 408K instead of ~3M.

---

Nice!